<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>319</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="run">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>120</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="start_time_check">
       <property name="text">
        <string>Start Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="end_time_check">
       <property name="text">
        <string>End Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="res_w_check">
       <property name="text">
        <string>Resolution Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="res_h_check">
       <property name="text">
        <string>Resolution Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bitrate_check">
       <property name="text">
        <string>Bitrate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="start_text"/>
     </item>
     <item>
      <widget class="QLineEdit" name="end_text"/>
     </item>
     <item>
      <widget class="QLineEdit" name="res_w_text"/>
     </item>
     <item>
      <widget class="QLineEdit" name="res_h_text"/>
     </item>
     <item>
      <widget class="QLineEdit" name="bitrate_text"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>59</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="input_file_button">
       <property name="text">
        <string>Select Input File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_file_text"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="volume_label">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="volume_slider">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLCDNumber" name="volume_number">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
